src/notifications.js

```diff
@@ -12,7 +12,7 @@ class Notifications extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const {notifications} = nextProps;
+    const notifications = nextProps.notifications.toJS();
```

Comment:
Hey ppl,

Was thinking about this, and I'm not sure whether we should force people to have immutable-js pulled down as a part of this lib.

What do you think?
Maybe we can make some sort of version which includes immutable as you suggested previously?

Thanks and apologies for late reply.